Tests/URKArchiveTestCase.m

```diff
@@ -31,6 +31,7 @@ - (void)setUp
     
     NSArray *testFiles = @[@"Test Archive.rar",
                            @"Test Archive (Password).rar",
+                           @"Test Archive (Bad Password).rar",
```

Comment:
As discussed above, there is no need to include a new archive, as the existing password-protected ones will work.